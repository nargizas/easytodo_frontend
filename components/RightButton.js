import { Pressable, View } from "react-native";
import { AntDesign } from '@expo/vector-icons';
import { GlobalStyles } from "../constants/styles";

function RightButton({ onPress }) {
    return <Pressable onPress={onPress}>
        <View>
            <AntDesign name="caretright" size={24} color="#f5f5f5" />
        </View>
    </Pressable>
}

export default RightButton;
