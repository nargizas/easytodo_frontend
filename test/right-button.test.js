import React from 'react';
import renderer from 'react-test-renderer';

import RightButton from '../components/RightButton';

it('renders correctly', () => {
  const tree = renderer.create(<RightButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
