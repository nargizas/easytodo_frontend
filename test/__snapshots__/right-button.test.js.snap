// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
>
  <View>
    <Text />
  </View>
</View>
`;
